src/test/components/TrackThread.test.js

```diff
@@ -188,4 +190,28 @@ describe('timeline/TrackThread', function() {
       selectionEnd: 2,
     });
   });
+
+  it('does not add disk io markers if none are present', function() {
+    const noMarkers = [];
+    const { container } = setup(getMarkersProfile(noMarkers));
+    expect(queryByTestId(container, 'TimelineMarkersDiskIo')).toBeFalsy();
```

Comment:
as said in the other PR, you can get `queryByTestId` from the result of `setup` and then you don't need to pass `container`.
(same below)